apps/financial-aid/web-veita/src/components/Profile/MunicipalityProfile.tsx

```diff
@@ -26,15 +26,15 @@ interface MunicipalityProfileProps {
 const MunicipalityProfile = ({ municipality }: MunicipalityProfileProps) => {
   const smallText = 'small'
   const headline = 'h5'
-  const aidTableBody = (value: AidType) => {
+  const aidTableBody = (value: AidTypeHomeCircumstances) => {
     switch (value) {
-      case AidType.OWNPLACE:
+      case AidTypeHomeCircumstances.OWNPLACE:
```

Comment:
Not to be addessed in this PR, but why do we need HomeCircumstances enum and then another for AidTypeHomeCircumstances, shouldn't these reflect each other?

Isn't the only difference that HomeCircumstances has an additional value of `OTHER` can't we just remove that?